.github/workflows/elixir.yml

```
@@ -15,9 +15,8 @@ jobs:
 
     strategy:
       matrix:
-        otp: [23.1, 22.3, 21.3]
-        elixir: [1.11, 1.10.4, 1.9.4, 1.8.2, 1.7.4]
-
+        otp: [24. 23, 22]
```

As per commit comment:

```suggestion
        otp: [24, 23, 22]
```

Sorry for that. 😅